jenkins/Jenkinsfile

```diff
@@ -279,9 +279,13 @@ pipeline {
           script {
             for(String dp_version: dataproc_versions.trim().split(' ')) {
               def dp_version_formatted = dp_version.replace(".","-")
+              // Dataproc 1.2 is used for the legacy image and must have python 3.6 installed
```

Comment:
This is a little hacky, but couldn't think of a cleaner way. Basically we have:
1. legacy image -- dataproc 1.2 -- python 3.6.8 on the workers
2. terra-docker image -- dataproc 1.4 -- python 3.7.4 on the workers